Kickstarter-iOS/Views/CommentComposerView.swift

```diff
@@ -144,6 +144,12 @@ final class CommentComposerView: UIView {
         self?.inputContainerView.inputTextView.resignFirstResponder()
       }
 
+    self.viewModel.outputs.updateTextViewHeight
+      .observeForUI()
+      .observeValues { [weak self] _ in
+        self?.inputContainerView.inputTextView.invalidateIntrinsicContentSize()
```

Comment:
Solution seems fine but I'm curious about one thing. Does resetInput() not achieve this for us?

Reply:
This line of code `self?.inputContainerView.inputTextView.invalidateIntrinsicContentSize()` was initially called in `textViewDidChange(_:)` delegate method but I wanted that to be controlled by a VM output and YES `self.viewModel.inputs.resetInput()` is one of the actions that triggers `self.viewModel.outputs.updateTextViewHeight`.